src/components/inputs/async-creatable-select-input/async-creatable-select-input.js

```diff
@@ -60,31 +99,27 @@ export class AsyncCreatableSelectInput extends React.Component {
         })
       ),
     ]),
-    onChange: PropTypes.func.isRequired,
-    onBlur: PropTypes.func,
     loadOptions: PropTypes.func.isRequired,
-    isMulti: PropTypes.bool,
+    cacheOptions: PropTypes.any,
 
-    components: PropTypes.object,
+    // Creatable props
+    allowCreateWhileLoading: PropTypes.bool,
     formatCreateLabel: PropTypes.func,
-    noOptionsMessage: PropTypes.func,
-    intl: PropTypes.shape({
-      formatMessage: PropTypes.func.isRequired,
-    }).isRequired,
-    hasError: PropTypes.bool,
-    hasWarning: PropTypes.bool,
+    isValidNewOption: PropTypes.func,
+    getNewOptionData: PropTypes.func,
+    onCreateOption: PropTypes.func,
+    createOptionPosition: PropTypes.string,
   };
 
   render() {
     return (
       <Constraints.Horizontal constraint={this.props.horizontalConstraint}>
         <div {...filterDataAttributes(this.props)}>
           <AsyncCreatableSelect
-            {...omit(this.props, [
-              'horizontalConstraint',
-              'hasError',
-              'hasWarning',
-            ])}
+            aria-label={this.props['aria-label']}
+            aria-labelledby={this.props['aria-labelledby']}
```

Comment:
maybe just use the `filterAriaAttributes` util?

Reply:
See https://github.com/commercetools/ui-kit/pull/120#discussion_r221872796